infra/enrolment-records/repository/src/test/java/com/simprints/infra/enrolment/records/repository/local/RoomEnrollmentRecordLocalDataSourceTest.kt

```diff
@@ -0,0 +1,1537 @@
+package com.simprints.infra.enrolment.records.repository.local
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import com.google.common.truth.Truth.assertThat
+import com.simprints.core.domain.face.FaceSample
+import com.simprints.core.domain.fingerprint.FingerprintSample
+import com.simprints.core.domain.fingerprint.IFingerIdentifier
+import com.simprints.core.domain.tokenization.TokenizableString
+import com.simprints.core.domain.tokenization.asTokenizableEncrypted
+import com.simprints.infra.config.store.models.Project
+import com.simprints.infra.enrolment.records.repository.domain.models.BiometricDataSource.Simprints
+import com.simprints.infra.enrolment.records.repository.domain.models.Subject
+import com.simprints.infra.enrolment.records.repository.domain.models.SubjectAction
+import com.simprints.infra.enrolment.records.repository.domain.models.SubjectQuery
+import com.simprints.infra.enrolment.records.room.store.SubjectsDatabaseFactory
+import com.simprints.infra.security.keyprovider.LocalDbKey
+import com.simprints.testtools.common.coroutines.TestCoroutineRule
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import kotlinx.coroutines.channels.toList
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.runTest
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import java.util.Date
+import java.util.UUID
+
+@RunWith(RobolectricTestRunner::class)
+class RoomEnrollmentRecordLocalDataSourceTest {
```

Comment:
The class (and the file) has double L while the source class has only a single one (as is the accepted naming convention).